### PR TITLE
Update the path for grafton-sanity-check.sh in hc.conf

### PR DESCRIPTION
**Pull request by dpmramesh**
Update the path for grafton-sanity-check.sh in hc.conf

### CODE REVIEW

**Comment by dpmramesh** [comment]
@sac Can you merge this pull request?

File: examples/hc.conf [hunk 1]
```diff
@@ -8,7 +8,7 @@ host3
 # Do a sanity check before proceding
 [script1]
 action=execute
-file=/usr/share/gdeploy/scripts/grafton-sanity-check.sh -d disk1,disk2 -h host1,host2
+file=/usr/share/ansible/gdeploy/scripts/grafton-sanity-check.sh -d disk1,disk2 -h host1,host2
```
Comment (sac) [inline examples/hc.conf:11]: @dpmramesh In master it has to be: /usr/share/ansible/gdeploy/scripts/grafton-sanity-check.sh

In 2.0.1 it is /usr/share/ansible/gdeploy that is going to change in 2.0.2

We are moving out of ansible directory.

**Review by sac — COMMENTED** [review]
Commented inline.

File: examples/hc.conf (hunk 1 above)
Comment (dpmramesh) [reply examples/hc.conf:11]: This is going to be tricky with oVirt releases. Currently I have added path usr/share/ansible/gdeploy/scripts/grafton-sanity-check.sh to my generated configuration. If you change this in future then we also have to change and release a new version. But ovirt release timeline may not align with gdeploy release time line.

Comment (dpmramesh) [reply examples/hc.conf:11]: @sac I would suggest you to take this changes in and change the path when you change the path in spec file.

**Pull request closed without merging**